src/pb_stub.cc

```diff
@@ -82,9 +82,10 @@ Stub::Instantiate(
     const std::string& shm_region_name, const std::string& model_path,
     const std::string& model_version, const std::string& triton_install_path,
     bi::managed_external_buffer::handle_t ipc_control_handle,
-    const std::string& name, const std::string& platform)
+    const std::string& name, const std::string& py_backend_based_model)
```

Comment:
Small nit: do we mean this to be `py_based_backend`?

Could we actually just use `backend` here for this variable - and always fill it in - meaning it would be `python` for custom models and `vllm` etc. for others?

Reply:
Yeah, names are subject to change=)

Reply:
The logic behind this variable is whether location to `model.py` is specified or not. If yes, then this variable holds the path. Just a context.